### ISSUE TITLE
case and shifted characters not changed in VMware player

### ISSUE THREAD

**Issue by GoogleCodeExporter** [eadevlin/autokey#84]
```
What steps will reproduce the problem?
1. run notepad in a VM using VMware player, workstation, etc.
2. try to use autokey to paste into that notepad

What is the expected output? What do you see instead?
instead of pasting completely normally, characters pasted are always lowercase 
and unshifted (; instead of :, or 5 instead of %)

Using other paste methods does not really work as the ctrl-v is also not 
detected properly.

What version of the product are you using? On what operating system?
opensuse 11.3, Qt version, 0.71.0

Please provide any additional information below.
I have not yet tried other virtualization platforms like virtualbox.

```

Original issue reported on code.google.com by `benk...@gmail.com` on 28 Sep 2010 at 3:02

**Comment by GoogleCodeExporter** [comment]
```
This method of using AutoKey is not supported - you must either run AutoKey 
inside the VM or if it is not a Linux VM use AutoHotKey or similar.
```

Original comment by `cdekter` on 28 Sep 2010 at 11:08
- Changed state: **Invalid**

**Comment by GoogleCodeExporter** [comment]
```
Should be fixed in v0.80.0
```

Original comment by `cdekter` on 29 Sep 2011 at 10:35
- Changed state: **Fixed**
